.github/workflows/signjars.yml

```
@@ -0,0 +1,83 @@
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+
+    - name: Codesign Internal Native Libraries
+      env: 
+        MACOS_CERTIFICATE: ${{ secrets.MACOS_CERTIFICATE }}
+        MACOS_CERTIFICATE_PWD: ${{ secrets.MACOS_CERTIFICATE_PWD }}
+        JARSIGNER_KEYSTORE_B64: ${{ secrets.JARSIGNER_KEYSTORE_B64 }}
+        JARSIGNER_STOREPASS: ${{ secrets.JARSIGNER_STOREPASS }}
+        JARSIGNER_ALIAS: ${{ secrets.JARSIGNER_ALIAS }}
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:6: Double quote to prevent globbing and word splitting

(shellcheck)